packages/ezresto-directory/ezresto-directory.0.2/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" "Ocamlpro" ]
+license: "LGPL-2.1-with-OCaml-exception"
+homepage: "https://gitlab.com/nomadic-labs/resto"
+bug-reports: "https://gitlab.com/nomadic-labs/resto/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/resto"
+synopsis: "A minimal OCaml library for type-safe HTTP/JSON RPCs"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" { >= "4.07" }
+  "dune" { >= "1.7" }
+  "ezresto" {= "0.2" }
```

Comment:
For next time be aware you can use ` {= version}` instead of `{= "0.2"}`

Reply:
Oh nice! I didn't know that trick!